src/components/overview/ExpandedIcons.jsx

```diff
@@ -45,33 +45,33 @@ export default function ExpandedIcons({ photos, currImgIdx, setCurrImgIdx }) {
         bottom: '2em',
       }}
     >
-      {currImgIdx !== 0 && (
-        <FontAwesomeIcon
-          icon={faCaretLeft}
-          size="lg"
-          color="black"
-          style={{
-            opacity: '0.5',
-          }}
-          onClick={() => {
+      <FontAwesomeIcon
+        icon={faCaretLeft}
+        size="lg"
+        color={currImgIdx !== 0 ? 'black' : 'white'}
```

Comment:
Hey you mentioned this as your solve, but just a heads up that doesn't work in dark mode.

Reply:
For sure, once we decide dark mode colors and have a dark mode global state I'm gonna have to modify the color logic a bit